CoreFramework/APICoreRestSharp/APIRequestRS.cs

```diff
@@ -0,0 +1,136 @@
+﻿//using CoreFramework.Reporter;
+//using RestSharp;
+//using RestSharp.Authenticators;
+//using RestSharp.Serialization;
+
+//namespace CoreFramework.APICoreRestSharp;
+
+/// APICore with RestSharp client
+
+//public class APIRequestRS
```

Comment:
Why do we need to comment on this class?
If you don't use it, please remove it. If not, then please write and test if these codes are working